doc/generate_doc.sh

```diff
@@ -2,6 +2,18 @@
 export WORK_ROOT=$(cd `dirname "$0"`;pwd)
 export BIGFLOW_PYTHON_HOME=$WORK_ROOT/../bigflow_python/python/
 BIGFLOW=$BIGFLOW_PYTHON_HOME/bigflow/bin/bigflow
+
+wget https://github.com/google/protobuf/releases/download/v2.5.0/protobuf-2.5.0.tar.gz -O protobuf-2.5.0.tar.gz && tar zxvf protobuf-2.5.0.tar.gz
```

Comment:
When we run this script on local machine, can we just use the existed directory if there is one.
Or you should delete the old one before download, in case we download a lot of protobuf-2.5.0.tar.gz.* .